services/etcd.go

```diff
@@ -7,15 +7,17 @@ import (
 	"github.com/docker/go-connections/nat"
 	"github.com/rancher/rke/docker"
 	"github.com/rancher/rke/hosts"
+	"github.com/rancher/rke/pki"
 	"github.com/rancher/types/apis/management.cattle.io/v3"
 	"github.com/sirupsen/logrus"
 )
 
 func RunEtcdPlane(etcdHosts []*hosts.Host, etcdService v3.ETCDService) error {
 	logrus.Infof("[%s] Building up Etcd Plane..", ETCDRole)
 	initCluster := getEtcdInitialCluster(etcdHosts)
-	for _, host := range etcdHosts {
-		imageCfg, hostCfg := buildEtcdConfig(host, etcdService, initCluster)
+	for i, host := range etcdHosts {
+		nodeName := fmt.Sprintf("%s-%d", pki.EtcdKeyCertPathPrefix, i)
```

Comment:
yea, this part, I've seen it in the code somewhere already :) https://github.com/rancher/rke/pull/174/files#diff-fc62d6e548ca9429f12a9edf0f3a8415R50 Consider extracting